zfit/models/kde.py

```diff
@@ -1363,7 +1371,7 @@ def __init__(self,
                 raise ValueError("obs can only be None if data is ZfitData with limits.")
             else:
                 obs = data.space
-        data, size, weights = self._convert_init_data_weights_size(data, weights, padding=padding)
+        data, size, weights = self._convert_init_data_weights_size(data, weights, padding=padding, limits=obs.limits)
```

Comment:
```suggestion
        data, size, weights = self._convert_init_data_weights_size(data, weights, padding=padding, limits=obs)
```

This should just be "obs", it's already a space, right? Or do I mix up something?

Reply:
I think its correct this way as padreflect wants to obtain the numerical limits not the space.

Reply:
Indeed, you're right!